src/eip4788_deployment/eip4788_deployment_launcher.star

```diff
@@ -0,0 +1,32 @@
+PYTHON_IMAGE = "python:3.11-alpine"
+EIP4788_DEPLOYMENT_SERVICE_NAME = "eip4788-contract-deployment"
+
+
+def deploy_eip4788_contract_in_background(plan, sender_key, receiver_key, el_uri):
```

Comment:
I wonder if this can be extracted to something common between this and `sender.py`;

Reply:
second thought, this script exit while the other script keeps running; maybe we can make the cmd

```
cmd=["/bin/sh", "-c", "pip install web3 && /tmp/sender.py"],
```

this will STOP the container when the script exits; the previous one needs the fancy no hup and such as that needs to keep running infinitely

Reply:
fixed, have a look

Reply:
tried it out, this approach blocks the exec phase until the contract deployment is done - which cannot happen until the chain has started. I'd prefer spinning it up and letting kurtosis' plan/exec phase finish and the deployment happens later on.

Reply:
oh alright, makes sense to me.